src/views/HomeView.vue

```diff
@@ -1,35 +1,57 @@
 <script setup lang="ts">
 import type { Question } from "@/types/index";
-import questionsJson from "@/assets/questions.json";
 import axios from "axios";
 import { useRoute } from "vue-router";
 import config from "@/config";
 import { ref } from "vue";
+import { store } from "@/store/index";
+import questionsJson from "@/assets/questions.json";
 
 let questions = ref(Array<Question>());
 
+let errorText = ref("");
+
+let isActive = false;
```

Comment:
```suggestion
const isActive = ref(false);
```
needs to be a ref, because state is changing during rendering